iOS-Email-Client/AppDelegate.swift

```diff
@@ -316,6 +325,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Use this method to release shared resources, save user data, invalidate timers, and store enough application state information to restore your application to its current state in case it is terminated later.
         // If your application supports background execution, this method is called instead of applicationWillTerminate: when the user quits.
         WebSocketManager.sharedInstance.pause()
+        passcodeLockPresenter.present()
```

Comment:
I don't know how this library works but the method `present()` at entering in background make sense?

Reply:
The point at this point was to overlay the current view controller with the pincode view controller so even if you see the miniature view in tasks, you wont se the app. This is changed in a future pr because of the free time period settings